colors/monokai.vim

```diff
@@ -31,6 +31,7 @@ hi VertSplit    guifg=#3B3A32 guibg=bg gui=none
 hi Folded       guifg=#75715E guibg=bg gui=none
 hi FoldColumn   guifg=#75715E guibg=#3E3D32 gui=none
 hi SignColum    guifg=#75715E guibg=#3E3D32 gui=none
+hi SignColumn   guifg=#75715E guibg=#3E3D32 gui=none
```

Comment:
Not sure about `SignColumn`, but before I added this line I had darker bg color rather than `LineNr`.